### PR TITLE
Module 5, Block 1 : Blog app - creating a data model

### PR DESCRIPTION
**Pull request by aceDavon**
## In this PR, I have implemented the requirements of the following task: :writing_hand: 

- [ ] Install Ruby on Rails framework.
- [ ] Use Ruby Gems as a software package system.
- [ ] Understand what ORM is.
- [ ] Use database migration files to maintain database schema.
- [ ] Use an ERD Diagram to create database migrations
- [ ] Add indexes to columns on the database tables where necessary as required by the project

### CODE REVIEW

File: db/migrate/20221122202416_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Database migration implementation
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.string :posts_counter
```
Comment (Munsa1) [inline db/migrate/20221122202416_create_users.rb]: - [ ] `posts_counter` should be of the data type integer, kindly fix this

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @aceDavon, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20221122202416_create_users.rb (hunk 1 above)
Comment (aceDavon) [reply db/migrate/20221122202416_create_users.rb]: Noted and resolved

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @aceDavon , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**